### PR TITLE
Add important tsconfig bases tsconfig.json.md

### PR DESCRIPTION
**Pull request by lopezac**
It's not necessary to add all TSConfigs bases possibles, but I think that Create React App and Next.js are important ones to highlight.
I added them in the order they were at the [tsconfig github repo](https://github.com/tsconfig/bases)

### CODE REVIEW

File: packages/documentation/copy/en/project-config/tsconfig.json.md [hunk 1]
```diff
@@ -95,6 +95,8 @@ For example, if you were writing a project which uses Node.js version 12 and abo
 This lets your `tsconfig.json` focus on the unique choices for your project, and not all of the runtime mechanics. There are a few tsconfig bases already, and we're hoping the community can add more for different environments.
 
 - [Recommended](https://www.npmjs.com/package/@tsconfig/recommended)
+- [Create React App](https://www.npmjs.com/package/@tsconfig/create-react-app)
+- [Next.js](https://www.npmjs.com/package/@tsconfig/next)
 - [Node 10](https://www.npmjs.com/package/@tsconfig/node10)
 - [Node 12](https://www.npmjs.com/package/@tsconfig/node12)
```
Comment (orta) [inline packages/documentation/copy/en/project-config/tsconfig.json.md]: May as well swap out these for the LTS version too

```suggestion
- [Node LTS](https://www.npmjs.com/package/@tsconfig/node-lts)
```

File: packages/documentation/copy/en/project-config/tsconfig.json.md [hunk 2]
```diff
@@ -95,8 +95,9 @@ For example, if you were writing a project which uses Node.js version 12 and abo
 This lets your `tsconfig.json` focus on the unique choices for your project, and not all of the runtime mechanics. There are a few tsconfig bases already, and we're hoping the community can add more for different environments.
 
 - [Recommended](https://www.npmjs.com/package/@tsconfig/recommended)
```
Comment (jakebailey) [inline packages/documentation/copy/en/project-config/tsconfig.json.md:97]: Is there any reason to keep this list at all when the intro statement just links to the repo?

**Review by jakebailey — CHANGES_REQUESTED** [review]
Yeah, I think it'd be better to just drop this list, which is just a duplicate of the readme in the tsconfig bases repo.

**Comment by jakebailey** [comment]
Opened #2845 to just remove the list.

**Pull request closed without merging**